test/test_core.py

```diff
@@ -200,3 +202,49 @@ def test_select_types(self, connection):
 
         row = connection.execute(sa.select(tb)).fetchone()
         assert row == (1, "Hello World!", 3.5, True, now, today)
+
+
+class TestUpsert(TablesTest):
```

Comment:
**issue (blocking)** Let's add also tests with several UPSERTs and UPSERT FROM SELECT, if it is supported, not sure